### ISSUE TITLE
Screen Reader Accessibility during login reads password out loud

### ISSUE THREAD

**Issue by garrettjwilke** [pop-os/libcosmic#1067]
when the Screen Reader Accessibility feature is enabled, if you attempt to log in all of your password keystrokes are read out loud.

tested this feature in gnome, and it masks the password and doesn't read out loud. also, in gnome if you click the little "show password" button, the screen reader will then start reading the input keystrokes.

unsure if this belongs here or another repo

**Comment by leviport** [comment]
It reads passwords from the pkexec dialog as well. Maybe this is either cosmic-text or libcosmic?

To clarify Gnome's behavior, when the password is hidden, it reads each character as "circle".

**Comment by WatchMkr** [comment]
Give the circle character to AccessKit (fix)
